### PR TITLE
rewrite switch section

### LINKED ISSUE #133: Adapt infra schema documentation for the track link deletion

### PR DESCRIPTION
**Pull request by younesschrifi**
Close #133

### CODE REVIEW

File: content/docs/explanation/models/data-models-full-example/index.en.md [hunk 1]
```diff
@@ -67,91 +67,115 @@ For all track sections in our infrastructure, the `geo` and `sch` attributes are
...
-`TrackSectionLinks` are used to connect two track sections together, just like a weld joint would in real life. In an OSRD simulation, a train can go from one track section to another only if they are connected by a `TrackSectionLink` (or by a `Switch`).
+#### Node
 
-To connect more than two `TrackSections` together, use the [`Switches`](#switches).
+A `Node` represents a node in the infrastructure. In an OSRD simulation, a train can only move from one section of track to another if they are linked by a node.
+A node can take one of two forms :
 
-In our infrastructure, since we chose to have our track sections as long as possible, we do not actually need to use `TrackSectionLinks`.
-`TrackSectionLinks` are always optional: two track sections connected by a link behave just like a single track section.
+**1) Switches**
```
Comment (Tguisnet) [inline content/docs/explanation/models/data-models-full-example/index.en.md:77]: We should remove distinction between links and switches

File: content/docs/explanation/models/data-models-full-example/index.en.md [hunk 2]
```diff
@@ -67,91 +67,115 @@ For all track sections in our infrastructure, the `geo` and `sch` attributes are
...
-Switches can be thought of as a collections of track section links, partitioned into groups. Each group represents for a switch state. Switching group can take time, and at most one link can be ready to use at a time.
+**2) Track section links**
 
-In the real world, switches are not unique, but rather instances of existing models.
-Thus, links and groups are not part of the switch itself, but in a `SwitchType` object, which is shared by switches of the same model.
+For the moment, we have only created track sections, which are not interconnected (geospatial data is not used to deduce which tracks are connected).
+
+Links are used to connect two track sections together, just as a weld joint would in real life. In an OSRD simulation, a train can only move from one section of track to another if they are connected by this type of node, the `link` (or by another `NodeType`).
+
+Whether for switches or track section links, links and groups are not part of the switch itself, but of a `Node` object, which is shared by switches in the same model.
```
Comment (Tguisnet) [inline content/docs/explanation/models/data-models-full-example/index.en.md:89]:
```suggestion
Whether for switches or track section links, links and groups are not part of the switch itself, but of a `NodeType` object, which is shared by switches in the same model.
```

File: content/docs/explanation/models/data-models-full-example/index.en.md [hunk 3]
```diff
@@ -67,91 +67,115 @@ For all track sections in our infrastructure, the `geo` and `sch` attributes are
...
 
-##### Switch Types
 
-`SwitchTypes` have two mandatory attributes:
+##### Node Types
+
+`NodeTypes` have two mandatory attributes:
 
 * `ports`: A list of port names. A port is an endpoint which can be connected to a track section.
 * `groups`: A mapping from group names to lists of links between two ports.
```
Comment (Tguisnet) [inline content/docs/explanation/models/data-models-full-example/index.en.md:97]: I should add a better explanation of those terms, maybe including the word `branch`

File: content/docs/explanation/models/data-models-full-example/index.en.md [hunk 4]
```diff
@@ -67,91 +67,115 @@ For all track sections in our infrastructure, the `geo` and `sch` attributes are
...
 
 A Point Switch only has two positions:
 
-![Point switch positions diagram](svg_diagrams/point_switch_positions.en.svg)
+- *A* to *B1*
+- *A* to *B2*
+
+![point switch position diagram](svg_diagrams/PointSwitch_AtoB1.svg) ![point switch position diagram](svg_diagrams/PointSwitch_AtoB2.svg)
 
 **2) The Cross Switch**
```
Comment (Tguisnet) [inline content/docs/explanation/models/data-models-full-example/index.en.md:125]:
```suggestion
**2) The Crossing**
```

File: content/docs/explanation/models/data-models-full-example/index.en.md [hunk 5]
```diff
@@ -67,91 +67,115 @@ For all track sections in our infrastructure, the `geo` and `sch` attributes are
...
 
-Here are the two different connections that this switch type has:
+- *A1* to *B1*
+- *A2* to *B2*
 
-![Cross switch positions diagram](svg_diagrams/cross_switch_positions.en.svg)
+
+![Cross Switch Diagram positions](svg_diagrams/Crossing_A1toB1.svg) ![Cross Switch Diagram positions](svg_diagrams/Crossing_A2toB2.svg)
 
 **3) The Double cross switch**
```
Comment (Tguisnet) [inline content/docs/explanation/models/data-models-full-example/index.en.md:143]:
```suggestion
**3) The Double slip switch**
```

File: content/docs/explanation/models/data-models-full-example/index.fr.md [hunk 6]
```diff
@@ -67,94 +67,135 @@ Pour toutes les sections de voies de notre infrastructure, les attributs `geo` e
...
+Les noms des ports doivent correspondre aux ports du type du noeud choisi. Les extrémités de la section de voie peuvent être début ou fin, faites attention à choisir les bonnes.
 
-La plupart des aiguillages de notre exemple sont des aiguillages habituels. Le chemin de la gare du Nord à la gare du Sud a deux aiguillages de croisement, et il y a un aiguillage de croisement double juste avant que la ligne principale ne se divise en lignes Nord-Est et Sud-Est.
+La plupart des noeuds de notre exemple sont des noeuds habituels. Le chemin de la gare du Nord à la gare du Sud a deux aiguillages de croisement, et il y a un aiguillage de croisement double juste avant que la ligne principale ne se divise en lignes Nord-Est et Sud-Est.
 
 ![Diagramme des sections de voie et des aiguillages](svg_diagrams/small_infra_rails_n_points.drawio.en.svg)
 
+Il est important de noter que ces noeuds sont présents en dur dans le code du projet. Seul les `extended_switch_type` ajoutés par l'utilisateur apparaîtront dans le railjson.
```
Comment (Tguisnet) [inline content/docs/explanation/models/data-models-full-example/index.fr.md:197]: I'll turn *en dur* by *par défaut* 
This part is not present in the English translation.

File: content/docs/explanation/models/data-models-full-example/index.fr.md [hunk 7]
```diff
@@ -67,94 +67,135 @@ Pour toutes les sections de voies de notre infrastructure, les attributs `geo` e
...
+![Diagramme d'un aiguillage de croisement simple](svg_diagrams/DoubleSlipCrossing_A1toB1.svg)
+
+Voici les trois connexions que peut réaliser cet aiguillage :
+
+- *A1* vers *B1*
+- *A1* vers *B2*
+- *A2* vers *B2*
+
+![Diagramme des positions de l'aiguillage de croisement simple](svg_diagrams/SingleSlipCrossing_A1toB1.svg) ![Diagramme des positions de l'aiguillage de croisement simple](svg_diagrams/SingleSlipCrossing_A1toB2.svg)
+![Diagramme des positions de l'aiguillage de croisement simple](svg_diagrams/SingleSlipCrossing_A2toB2.svg)
```
Comment (Tguisnet) [inline content/docs/explanation/models/data-models-full-example/index.fr.md:173]: This part is not present in the English translation.

**Review by Tguisnet — CHANGES_REQUESTED** [review]
Good work thanks ! I request few changes.

**Review by multun — CHANGES_REQUESTED** [review]

**Pull request merged**